examples/vue/src/stories/index.js

```diff
@@ -96,4 +98,15 @@ storiesOf('Addon Notes', module)
   ))
   .add('with some bold text', withNotes({ notes: 'My notes on some bold text' })(() => ({
     render: h => h('div', [h('strong', ['A very long text to display'])])
-  })));
\ No newline at end of file
+  })));
+
+
+  storiesOf('Addon Knobs', module)
+    .addDecorator(withKnobs)
```

Comment:
I think we're going to need to expand the decorator API for this kind of thing (hopefully in a backwards compatible way)